scripts/js/lib/api/releaseNotes.ts

```diff
@@ -23,6 +23,9 @@ import { C_API_BASE_PATH, DOCS_BASE_PATH } from "./conversionPipeline.js";
 import { kebabCaseAndShortenPage } from "./normalizeResultUrls.js";
 import { removePrefix } from "../stringUtils.js";
 
+import _ from "lodash";
```

Comment:
You should instead import `lodash-es` like this

https://github.com/Qiskit/documentation/blob/850ee46526d78789703fbf1c576fae0a5029ca09/scripts/js/lib/stringUtils.ts#L13